index.js

```diff
@@ -40,6 +43,8 @@ async function analyzeCommits(pluginConfig, context) {
         return true;
       })
       .map(({message, ...commitProps}) => ({rawMsg: message, message, ...commitProps, ...parser(message, config)}))
+      // Filter commits based on releaseScopes
+      .filter(({scope}) => (releaseScopes.length > 0 ? releaseScopes.includes(scope) : true))
```

Comment:
```suggestion
      .filter(({scope}) => !!releaseScopes.length && releaseScopes.includes(scope))
```

Reply:
![image](https://user-images.githubusercontent.com/2154296/165104682-8a785483-9c79-4256-89c2-c66575b7d132.png)
not exactly

Reply:
Arf, je l'ai faite à l'envers :

```suggestion
      .filter(({scope}) => !releaseScopes.length || releaseScopes.includes(scope))
```